### PR TITLE
Small changes... 

### PR DESCRIPTION
**Pull request by chelsea-EYDS**
- added fiscal date
- removed location view
- migrations/ queries to insert the master location data and payment data

### CODE REVIEW

File: apps/backend/src/database/migrations/1674514238005-migration.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { locationQuery } from '../queries/locations';
```
Comment (vyasworks) [inline apps/backend/src/database/migrations/1674514238005-migration.ts:2]: Let's exclude the data push via a migration for now.

File: apps/backend/src/database/queries/locations.ts [hunk 2]
```diff
@@ -0,0 +1,376 @@
+export const locationQuery = `INSERT INTO public.master_location_data (id,"GARMS Location","Type","Location",description,"Program","Program Description","Min Client","Responsibility Code","Service Line",stob,"Project No.","Merchant ID",notes) VALUES('96a8a191-efe7-47e6-9a29-1392f6f4ce60',61,'Bank',20061,'100 MILE HOUSE',70,'SERVICE BC                              ',74,'32H61',58200,1461,3200000,'Bank',' '),
```
Comment (vyasworks) [inline apps/backend/src/database/queries/locations.ts:1]: remove this file

**Review by vyasworks — CHANGES_REQUESTED** [review]

File: apps/backend/src/temp-scripts/sftp.garms.sh [hunk 3]
```diff
@@ -9,7 +9,7 @@ files="transaction/*.JSON"
```
Comment (vyasworks) [inline apps/backend/src/temp-scripts/sftp.garms.sh:9]: we can remove this file, now that we have rclone impl

File: apps/backend/src/temp-scripts/sftp.bcm.sh [hunk 4]
```diff
@@ -26,15 +26,15 @@ sleep 3
 TDI34="TDI34/*.TXT"
 for i in ${TDI34[@]}
 do  
-    awslocal s3api put-object --bucket bc-pcc-data-files-local --key $i --body $i --metadata "type=tdi_34"
```
Comment (vyasworks) [inline apps/backend/src/temp-scripts/sftp.bcm.sh:29]: we can remove this file, now that we have rclone impl

**Comment by chelsea-EYDS** [comment]
Will reopen after syncing local

**Pull request closed without merging**